lib/active_model_serializers/adapter/attributes.rb

```diff
@@ -4,7 +4,9 @@ class Attributes < Base
       def serializable_hash(options = nil)
         options = serialization_options(options)
         options[:fields] ||= instance_options[:fields]
-        serializer.serializable_hash(instance_options, options, self)
+        serialized_hash = serializer.serializable_hash(instance_options, options, self)
```

Comment:
(This happens when a `CollectionSerializer` is used, which maps the resources into an array)